### PR TITLE
Add leaderboard entries for global and ladder_1v1.

### PR DESCRIPTION
**Pull request by cleborys**
Populate `leaderboard` with rating types for global and ladder 1v1, see #197.

Necessary for https://github.com/FAForever/server/pull/553 / https://github.com/FAForever/server/issues/534.
 
I hope I am using `name_key` and `description_key` as intended.

### CODE REVIEW

File: migrations/V84_add_leaderboard_entries.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+INSERT INTO leaderboard (id, technical_name, name_key, description_key) VALUES
+  (1, 'global', 'leaderboard.global.name', 'leaderboard.global.description'),
+  (2, 'ladder_1v1', 'leaderboard.ladder_1v1.name', 'leaderboard.ladder_1v1.description'),
+;
+
+INSERT INTO messages (key, language, region, value) VALUES
```
Comment (norraxx) [inline migrations/V84_add_leaderboard_entries.sql]: BTW: where do we use localization?

**Comment by norraxx** [comment]
For me seems good

File: migrations/V84_add_leaderboard_entries.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+INSERT INTO leaderboard (id, technical_name, name_key, description_key) VALUES
+  (1, 'global', 'leaderboard.global.name', 'leaderboard.global.description'),
```
Comment (Askaholic) [inline migrations/V84_add_leaderboard_entries.sql]: I'm not entirely sure how the i18n keys are supposed to work. I didn't even know there was a `messages` table, I assumed these keys would map to a file in the client somewhere. Maybe @Brutus5000 knows.

Comment (Brutus5000) [reply migrations/V84_add_leaderboard_entries.sql]: https://github.com/FAForever/db/blob/develop/migrations/V1__base.sql#L1053

But it's not that well maintained.

**Review by Brutus5000 — CHANGES_REQUESTED** [review]
The file needs a double underscore after the version number.

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**